packages/dependency-extraction-webpack-plugin/lib/util.js

```diff
@@ -3,6 +3,7 @@ const BUNDLED_PACKAGES = [
 	'@wordpress/icons',
 	'@wordpress/interface',
 	'@wordpress/undo-manager',
+	'@wordpress/sync',
```

Comment:
bundling sync package is only necessary if the "tree shaking didn't exclude the package somehow" on Core (no white page).

If we're still experiencing issues, we can consider bundling. There are two files to update, this one and `tools/webpack/packages.js`

Reply:
Tree shaking probably doesn't work in the `development` mode, so this might behave strangely for the unminified version of the `wp-core-data` script when not using bundling. I haven't tested, but I guess that we would see `wp.sync` global replacing the import and the referenced code would never run because it's behind the feature flag. So the only issue would be really the `wp-sync` dependency listed in the asset file. Well, we could always register that entry point in the dev mode and it shouldn't be a big deal 🤷🏻 

It definitely needs some testing as this is quite advanced usage. If @youknowriad has some recommendation, I would follow his guidance, as he knows best what is the desired configuration in the Gutenberg plugin.

Reply:
My hesitation is whether the dependency extraction happens before or after tree shaking. If it happens before, we're screwed and we need to "bundle" (but it's fine because the bundled code is going to be tree shaken in core anyway), but if the webpack dependency extraction happens after tree shaking, then the dependency to wp-sync won't be added in core, so everything would work.

Reply:
Is there any downside at all to trying to bundle now? My concern is that Beta 1 is tomorrow and the package update in core is pending. Would be good to get this sorted out now 😅

Reply:
@tellthemachines No big downsides no, the only one I see is that some people could have started using the wp-sync global in WordPress and bundling would break that.

Reply:
Good point, but given the sync package is still highly experimental that doesn't seem a huge risk. I'd say it's worth it to mitigate core update issues for this release 😅

Reply:
If that helps move things forward, it's also possible to land these changes only in `wp/6.4` branch.